src/components/button/button.module.scss

```diff
@@ -2,14 +2,14 @@
 .request {
   background-color: black;
   height: 70px;
-  width: 10%;
+  width: 80%;
   border: none;
   cursor: pointer;
-  border-radius: 0 10px 10px 0;
+  border-radius: 0 0 10px 10px;
 
-  @media ($mobil) {
-    border-radius: 0 0 10px 10px;
-    width: 80%;
+  @media ($mobile) {
+    border-radius: 0 10px 10px 0;
+    width: 20%;
```

Comment:
Your button looks ugly. There is no need to use percentage here. You can set regular pixels here.
<img width="1416" alt="image" src="https://github.com/michey85/IPAdress/assets/36237995/030fbe78-35ae-4d4d-bccc-2bc23a0efa91">